lib/Github/Api/Repo.php

```diff
@@ -793,4 +793,20 @@ public function transfer($username, $repository, $newOwner, $teamId = [])
     {
         return $this->post('/repos/'.rawurldecode($username).'/'.rawurldecode($repository).'/transfer', ['new_owner' => $newOwner, 'team_id' => $teamId]);
     }
+
+    /**
+     * Create a repository using a template.
+     *
+     * @link https://developer.github.com/v3/repos/#create-a-repository-using-a-template
+     *
+     * @param string $templateOwner
+     * @param string $templateRepo
+     * @param array  $parameters
+     *
+     * @return array
+     */
+    public function createFromTemplate($templateOwner, $templateRepo, array $parameters = [])
```

Comment:
Can you also add a `string` typehint to both parameters and remove the superfluous `@param` docs?